packages/teams-js/src/public/interfaces.ts

```diff
@@ -749,6 +749,43 @@ export interface DeepLinkParameters {
   subEntityWebUrl?: string;
 }
 
+/**
+ * Shared Dialog Properties
+ */
+export interface BaseDialogInfo {
```

Comment:
Since we are adding support for Card and Bot, does it make sense to add depreciate tag to the original dialogInfo interface?

Reply:
It depends on do we want to update it across the platform, and if so, why? Also, my point of view is we need to add all these interfaces back up when are removing the support for `DialogInfo`

Reply:
Because the only place where `DialogInfo` interface is being used is in tasks module and that is deprecated. 

What do you mean by this: "we need to add all these interfaces back up when are removing the support for DialogInfo"

Reply:
I think we do use it in `dialog.adaptiveCards.open` 
`const dialogInfo: DialogInfo = getDialogInfoFromAdaptiveCardDialogInfo(adaptiveCardDialogInfo);`

Reply:
I will get @TrevorJoelHarris 's view on this as well :)

Reply:
(We should also update the comment on TaskInfo to point to UrlDialogInfo and the other ones now

Reply:
hidden sounds good! If we ever plan to change the hubSDK interfaces to match the interfaces in appSDK (for simplicity?), we can come back to this :)